hildon/hildon-touch-selector-entry.c

```diff
@@ -424,6 +429,9 @@ hildon_touch_selector_entry_set_input_mode (HildonTouchSelectorEntry * selector,
   priv = HILDON_TOUCH_SELECTOR_ENTRY_GET_PRIVATE (selector);
 
   hildon_gtk_entry_set_input_mode (GTK_ENTRY (priv->entry), input_mode);
+#else
+  g_warning("hildon_gtk_entry_set_input_mode unsupported");
```

Comment:
This warning differs from others. Please keep the same style. I mean "no-op" part.
Either remove it or use it the same way everywhere.

Reply:
Fixed